cdap-runtime-spi/pom.xml

```diff
@@ -35,6 +35,11 @@
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
     </dependency>
+    <dependency>
+      <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-api</artifactId>
```

Comment:
We should not add cdap-api as a dependency. Ignore my previous comment about linking to Requirements in the Provisioner API.